src/interactive-window/editor-integration/codeGenerator.ts

```diff
@@ -364,15 +297,13 @@ export class CellHashProvider implements ICellHashProvider {
     where we are making debugging calculations for runtimeLine feels appropriate.
     */
     private addHiddenLines(
+        debug: boolean,
         cell: NotebookCell,
         source: string[],
         trueStartLine: number,
         firstNonBlankLineIndex: number
     ): { runtimeLine: number; debuggerStartLine: number } {
-        if (
-            this.debugService?.activeDebugSession &&
```

Comment:
I think this class needs to be told whether it should generate debugging code or not.
There by removes the debug service and the place where we call the method `generateCode` (previously called `addCellhash`) we already have a variable named `isDebug`).